packages/react-core/src/components/TextInputGroup/__tests__/TextInputGroupMain.test.tsx

```diff
@@ -180,7 +181,7 @@ describe('TextInputGroupMain', () => {
     expect(hintInput).toBeInTheDocument();
   });
 
-  it('renders the hint input with classes pf-v5-c-text-input-group__text-input and pf-m-hint', () => {
+  it(`renders the hint input with classes ${styles.textInputGroupTextInput} and pf-m-hint`, () => {
```

Comment:
should this use the `styles.modifiers` obj?

```suggestion
  it(`renders the hint input with classes ${styles.textInputGroupTextInput} and ${styles.modifiers.hint}`, () => {
```